_includes/head.html

```diff
@@ -4,6 +4,7 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
 	<!--[if lte IE 8]><script src="{{ "assets/js/ie/html5shiv.js" | relative_url }}"></script><![endif]-->
 	<link rel="stylesheet" href="{{ "assets/css/main.css" | relative_url }}" />
+	<script defer src="{{ "assets/js/all.js" | relative_url }}"></script>
```

Comment:
Not sure why `all.js` is necessary as I am not very knowledgable about fontawesome. Is there a way to use it without this, seems like a big dependency?

Reply:
Also the name `all.js` is kind of ambigious.

Reply:
For FA v5 we have a few options to serve it. Please check this [link](https://fontawesome.com/how-to-use/on-the-web/setup/hosting-font-awesome-yourself). We can prefer CSS alternative but it needs more files in total 2.8 MB. I know that 1MB is also huge, but this is the quickest way to use this icons. Alternatively, CDN can be used. Please check this [link](https://blog.fontawesome.com/introducing-font-awesome-kits/). However, imho, maintainer should be you for further changes of icons. And name can be changed it is not clear I agree :) I did not notice while I was following the instructions in the first link :)

Reply:
I see, I am not sure how much of a burden it is to host all the icons but probably not much. I think the ideal would be to use CDN and make a kit specific for the website. I am curious to see what @drac thinks about this but I am fine with merging with only changing `all.js` to something like `fa.js`. Thanks for the links btw!

Reply:
"all.js" renamed to "fontawesome_v5.10.1.js". I guess this is mot understandable name for it :) You are welcome ^^